src/Servers/HttpSys/test/FunctionalTests/ServerTests.cs

```diff
@@ -366,31 +366,6 @@ public void Server_SetConnectionLimitArgumentValidation_Success()
             }
         }
 
-        [ConditionalFact]
-        public async Task Server_SetConnectionLimit_Success()
-        {
-            using (Utilities.CreateDynamicHost(out var address, options =>
-            {
-                Assert.Null(options.MaxConnections);
-                options.MaxConnections = 3;
-            }, httpContext => Task.FromResult(0)))
-            {
-                using (var client1 = await SendHungRequestAsync("GET", address))
-                using (var client2 = await SendHungRequestAsync("GET", address))
-                {
-                    using (var client3 = await SendHungRequestAsync("GET", address))
-                    {
-                        // Maxed out, refuses connection and throws
-                        await Assert.ThrowsAsync<HttpRequestException>(() => SendRequestAsync(address));
-                    }
-
-                    // A connection has been closed, try again.
-                    string responseText = await SendRequestAsync(address);
```

Comment:
Would this test be reliable if we removed just this last request? Looking at the [last failure](https://dnceng.visualstudio.com/internal/_build/results?buildId=397114&view=ms.vss-test-web.build-test-results-tab&runId=12397114&resultId=119016&paneView=debug), that seems to be where the problem is.

Do we have any other tests that verifies `HttpSysOptions.MaxConnections` actually does anything?

Reply:
If we stopped passing the option to HttpSys, would any tests fail other than this one we're removing?

Reply:
No.

Reply:
> Do we have any other tests that verifies `HttpSysOptions.MaxConnections` actually does anything?

Yes